src/Header/DownloadStream.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef GROUP4_HEADER_DownloadStream_H_
+#define GROUP4_HEADER_DownloadStream_H_
+
+#include <string>
+
+// read file from local
+class DownloadStream
+{
+public:
+    DownloadStream();
+    // filename will include file path, if run successfully, it will return 1, else 0;
+    int readFile(std::string filename, std::string &binary);
```

Comment:
const ref.

Reply:
`readNextChunk()`?

Reply:
Since using const ref of string pointer is non-intuitive,  so I choose const pointer.